linera-core/src/unit_tests/test_helpers.rs

```diff
@@ -1,57 +1,90 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-#[cfg(test)]
-use assert_matches::assert_matches;
 use linera_chain::{ChainError, ChainExecutionContext};
 use linera_execution::{ExecutionError, SystemExecutionError};
 
 use crate::{client::ChainClientError, local_node::LocalNodeError, worker::WorkerError};
 
-#[cfg(test)]
+/// Asserts that an error is due to insufficient funding during an operation.
 pub fn assert_insufficient_funding_during_operation<T>(
     obtained_error: &Result<T, ChainClientError>,
```

Comment:
Idea: we could probably also take ownership of this parameter. This might simplify the code as well with less borrows and dereferences.